src/app/utils/animations/useSkeletonAnimation.tsx

```diff
@@ -26,11 +26,13 @@ export function useSkeletonAnimation({
     }
   }, [])
 
-  const animatedStyle = useAnimatedStyle(() => ({
-    opacity: enableSkeletonAnimation
-      ? interpolate(shared.value, [0, 1], [targetOpacityValue, 1])
-      : 1,
-  }))
+  const animatedStyle = useAnimatedStyle(() => {
+    return {
+      opacity: enableSkeletonAnimation
+        ? interpolate(shared.value, [0, 1], [targetOpacityValue, 1])
+        : 1,
+    }
```

Comment:
I remember in one of the initial fixes I did there was also an issue coming from something like [this](https://github.com/artsy/eigen/pull/8513/files), what do you think of changing this to:

```suggestion
    if (enableSkeletonAnimation) {
      return {
        opacity: interpolate(shared.value, [0, 1], [targetOpacityValue, 1])
      }
    }
    return {
      opacity: 1
    }
```

just to be on the safe side of things